src/utils/color.ts

```diff
@@ -0,0 +1,7 @@
+import hexRgb from 'hex-rgb';
+
+export const getRgbaFromHex = (hex: string, opacity: number) => {
```

Comment:
should we change other usages as well to call this?

Reply:
Hm, there's a fair number of these that could be changed over. I'll do them in another PR

Reply:
Create a ticket for tracking?

Reply:
https://terraformation.atlassian.net/browse/SW-3736

Reply:
Awesome, thanks !!